validator/src/bootstrap.rs

```diff
@@ -526,18 +538,21 @@ pub fn rpc_bootstrap(
         }
 
         while vetted_rpc_nodes.is_empty() {
-            let rpc_node_details_vec = get_rpc_nodes(
+            let rpc_node_details = match get_rpc_nodes(
                 &gossip.as_ref().unwrap().0,
                 cluster_entrypoints,
                 validator_config,
                 &mut blacklisted_rpc_nodes.write().unwrap(),
                 &bootstrap_config,
-            );
-            if rpc_node_details_vec.is_empty() {
-                return;
-            }
+            ) {
+                Ok(rpc_node_details) => rpc_node_details,
+                Err(err) => {
+                    error!("Failed to get RPC nodes: {}", err);
```

Comment:
```suggestion
                    error!("Failed to get RPC nodes: {err}
                           Consider removing `--no-port-check` or adjusting `--known-validator ...` 
                           arguments as applicable");
```

let's offer some guidance. (prolly wanna check how this looks in the log)

Reply:
Spun up a validator with a broken system clock and confirmed we print this message and then exit
```
[2023-02-15T17:36:38.383484109Z ERROR solana_validator::bootstrap] Failed to get RPC nodes: Unable to find any RPC peers. Consider checking system clock, removing `--no-port-check`, or adjusting `--known-validator ...` arguments as applicable
```